kroxylicious-integration-tests/src/test/java/io/kroxylicious/proxy/testplugins/package-info.java

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+@ReturnValuesAreNonnullByDefault
+@DefaultAnnotationForParameters(NonNull.class)
+@DefaultAnnotation(NonNull.class)
+package io.kroxylicious.proxy.testplugins;
```

Comment:
Worth noting that I'm putting these filters which we're using for testing into a separate package now. Felt a bit cleaner for files to be in a different package than any of the API or implementation (where we might inadvertantly access package-access members). Assuming this gets accepted I will make a PR to move the existing test filters here too.